styles.css

```diff
@@ -603,3 +602,64 @@ button:active {
   background-color: #2230d2;
   color: white;
 }
+
+.contact-form {
+  min-height: 718px;
+  background-image: url("Image/background-Contact-me.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.title-contact {
+  padding-top: 40%;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+}
+
+.view {
+  padding-top: 30px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 20px;
+  text-align: center;
+  color: #ebebff;
+  margin-right: 10%;
+  margin-left: 5%;
+  padding-bottom: 10%;
+}
```

Comment:
### OPTIONAL
- [ ] For DRY CSS you should combine duplicate properties. For example, you can create a class `.font-family-poppins{ font-family: Poppins}` and use this class in an HTML file.

- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)